config/projects/chef.rb

```diff
@@ -32,7 +32,7 @@
   install_dir "#{default_root}/#{name}"
 end
 
-override :bundler,        version: "1.10.7.depsolverfix.0"
+override :bundler,        version: "1.10.6"
```

Comment:
There is a bug in 1.10.6 which caused us problems when we were releasing 12.5.x because of some circular dependency issues with ohai.  I thought my branch fixed that issues...  but it didn't really fix it in all situations - just made it a bit better for the few we ran into.  We aren't running into those issues now and we already rolled it back in chefdk.  We just never got around to doing it in chef-client.